framework/test/util/TestUtilFrameworkListener.cpp

```diff
@@ -57,16 +55,16 @@ bool TestFrameworkListener::CheckEvents(
         i < events.size() ? events[i] : FrameworkEvent();
       const FrameworkEvent& pR =
         i < _events.size() ? _events[i] : FrameworkEvent();
-      US_TEST_OUTPUT(<< " - " << pE << " - " << pR);
+      std::cout<< " - " << pE << " - " << pR;
     }
   } else {
     for (std::size_t i = 0; i < events.size(); ++i) {
       const FrameworkEvent& pE = events[i];
       const FrameworkEvent& pR = _events[i];
       if (pE.GetType() != pR.GetType() || pE.GetBundle() != pR.GetBundle()) {
         listenState = false;
-        US_TEST_OUTPUT(<< "*** Wrong framework event ***\n found " << pR
-                       << "\n expected " << pE);
+        std::cout<< "*** Wrong framework event ***\n found " << pR
+                       << "\n expected " << pE;
       }
```

Comment:
Is it possible to use google test Marco for validation?

Reply:
It is possible.
However, these files are being used by tests in 'framework/test/driver' as well. 
Once all these tests are migrated, I can make changes to 'util' files to make it use gtest macros.